modules/local/publish_decontaminated.nf

```diff
@@ -0,0 +1,13 @@
+process PUBLISH_DECONTAMINATED {
+
+    input:
+    tuple val(meta), path(decontaminated_contigs)
+
+    output:
+    path("decontaminated.txt")
+
+    script:
+    """
+    mv ${decontaminated_contigs} decontaminated.txt
```

Comment:
could you do mv ${decontaminated_contigs} ${meta.id}.txt

Reply:
yes